content/github-actions/runner-types.mdx

```diff
@@ -14,37 +14,40 @@ jobs:
 
 **Note**: We support the `depot-ubuntu-latest-*` alias for `depot-ubuntu-22.04-*` if you want to use the latest Ubuntu version.
 
+**RAM Disk Reserved**:
+Most Depot runners have a portion of memory reserved for a RAM disk. The RAM disk cache acts as buffer before writing to the NVMe SSD for incredibly fast I/O operations.
+
 ## Intel runners
 
 Intel runners use AMD EC2 instances. Their EBS volume is provisioned with 8000 IOPS and 250 MB/s throughput.
 
 The following labels are available:
 
-| Label                      | CPUs | Memory | Disk size | Per-minute price |
-| :------------------------- | :--- | :----- | :-------- | :--------------- |
-| `depot-ubuntu-22.04-small` | 2    | 2 GB   | 100 GB    | $0.002           |
-| `depot-ubuntu-22.04`       | 2    | 8 GB   | 100 GB    | $0.004           |
-| `depot-ubuntu-22.04-4`     | 4    | 16 GB  | 150 GB    | $0.008           |
-| `depot-ubuntu-22.04-8`     | 8    | 32 GB  | 300 GB    | $0.016           |
-| `depot-ubuntu-22.04-16`    | 16   | 64 GB  | 600 GB    | $0.032           |
-| `depot-ubuntu-22.04-32`    | 32   | 128 GB | 1200 GB   | $0.064           |
-| `depot-ubuntu-22.04-64`    | 64   | 256 GB | 2400 GB   | $0.128           |
+| Label                      | CPUs | Memory | Disk size | Per-minute price | RAM Disk Reserved |
```

Comment:
I think I'd call this `Disk accelerator size` and probably move it so that it's after Disk size and before price (basically keeping price last in the list)